share/instruments/instrumentsxml.h

```diff
@@ -449,14 +449,15 @@ QT_TRANSLATE_NOOP("InstrumentsXML", "Horn in A♭"),
 QT_TRANSLATE_NOOP3("InstrumentsXML", "A♭ Hn.", "Horn in A♭"),
 QT_TRANSLATE_NOOP("InstrumentsXML", "Horn in G"),
 QT_TRANSLATE_NOOP3("InstrumentsXML", "G Hn.", "Horn in G"),
-QT_TRANSLATE_NOOP("InstrumentsXML", "E Horn"),
-QT_TRANSLATE_NOOP3("InstrumentsXML", "E Hn.", "E Horn"),
+QT_TRANSLATE_NOOP("InstrumentsXML", "Horn in E"),
+QT_TRANSLATE_NOOP3("InstrumentsXML", "E Hn.", "Horn in E"),
 QT_TRANSLATE_NOOP("InstrumentsXML", "Horn in E♭"),
 QT_TRANSLATE_NOOP3("InstrumentsXML", "E♭ Hn.", "Horn in E♭"),
 QT_TRANSLATE_NOOP("InstrumentsXML", "Horn in D"),
-QT_TRANSLATE_NOOP3("InstrumentsXML", "Hn.", "Horn in D"),
+QT_TRANSLATE_NOOP3("InstrumentsXML", "D Hn.", "Horn in D"),
 QT_TRANSLATE_NOOP("InstrumentsXML", "Horn in C"),
 QT_TRANSLATE_NOOP3("InstrumentsXML", "C Hn.", "Horn in C"),
+QT_TRANSLATE_NOOP("InstrumentsXML", "Horn in C (Treble Clef)"),
```

Comment:
We probably need a `QT_TRANSLATE_NOOP3("InstrumentsXML", "C Hn.", "Horn in C (Treble Clef)",` added here

Reply:
No, sorry, not needed

Reply:
Well it is needed, but with duplicating the 2 lines aabobe and another added and a changed lione in the other file. See #5232